src/components/UI/Card/ProductCards/ProductCards.jsx

```diff
@@ -16,100 +18,169 @@ import { IcTwotoneShoppingCart } from "../../Icons/IcTwotoneAddShoppingCart";
 // stock={item.stock}
 // categoryType={item.categoryType}
 function ProductCards(props) {
-    const { title,productImage,author,price,discount,discountedPrice,rating,productType,feature,stockType,stock,categoryType} =props;
-    const {cartState,setCartState}= useCart();
-    const {WishListState,setWishListState} =useWishList();
-    console.log(arrivalBanner5);
-    const AddProductsInCartHandler = async (item)=>{
-        try{
+    const {
+        _id,
+        title,
+        productImage,
+        author,
+        price,
+        discount,
+        discountedPrice,
+        rating,
+        productType,
+        feature,
+        stockType,
+        stock,
+        categoryType
+    } = props;
+    const {cartState, setCartState} = useCart();
+    const {WishListState, setWishListState} = useWishList();
+    const [handleCartQuantity, setHandleCartQuantity] = useState();
+    let holder = true;
+    console.log(cartState);
+    const AddProductsInCartHandler = async (item) => {
+        try {
             console.log(item);
-            await axios.post("/api/user/cart",
-                    { "product":item},
-                    { headers:
-                        {
-                            authorization:localStorage.getItem("feetz")
-                        }
-                    })
-                        .then((res)=>{
-                            console.log(res);
-                            if(res.status===201){
-                                setCartState(res.data.cart);
-                            }
-                        })
-                        .catch((error)=>{
-                            console.log(error.message);
-                        });
-        }
-        catch(err){
-            console.log("error ",err.message);
+            await axios.post("/api/user/cart", {
+                "product": item
+            }, {
+                headers: {
+                    authorization: localStorage.getItem("feetz")
+                }
+            }).then((res) => {
+                console.log(res);
+                if (res.status === 201) {
+                    setCartState(res.data.cart);
+                }
+            }).catch((error) => {
+                console.log(error.message);
```

Comment:
You are handling the promise first using Async then you are handling it with the .then property you could have used only the async or the traditional .then.

Reply:
ok